modules/programs/irssi.nix

```diff
@@ -0,0 +1,213 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.irssi;
+
+  ifToYesNo = b: if b then "yes" else "no";
+
+  assignFormat =
+    set:
+      concatStringsSep "\n"
+        (
+          mapAttrsToList (k: v: "  ${k} = \"${v}\";") set
```

Comment:
Perhaps `v` needs to be escaped here in case it contains " or something?

Reply:
Yes, and I noticed that some other places need to be escaped, too (namely `autoCommand`). I added escaping to the nick, too, but a quote in the nick is invalid for IRC nicks.

Reply:
Nickname checking is probably out of scope, as the allowed charset differs between servers (mostly because of UTF-8).